.travis.yml

```diff
@@ -5,4 +5,21 @@ compiler:
 - gcc
 - clang
 
-script: ./autogen.sh && make 
+env:
+  global:
+   # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
+   #   via the "travis encrypt" command using the project repo's public key
+   - secure: "kfVYmDS4SJPlDVZM++jfkBpRq/ET/EFn6kaYlWnW3udZ7Xpqr7DC9DyUa55gil83M+TjBMCjaGL4jQD7TREywohnYcy6134CwZwhCGZpFGioJVv4OH2Np5G8HFHId26qrtllC7lv0J97ZBmGCs7muP27h6HfDeDCavbHGcrjH+1x4uTi/0M7a5ABtvzWE/JspUNN84Es9ARSkCOqNKjdTT8D/QgcnVQ3a0VHJvk7q/1ZLeRf9D28rlpv7bSC/+kUDRul63hYCqnLW0dFuvx8idlDQ9S2Q9G0vxY8w9rSjGhJ7z8OvV/s4quBEnbFnE7YIhlUKsI/ouF8zRHDb+h8pmT4eIq137RKEt2l8Uee2vjVnbKbCEMW52VggO/NM+8Zu1tFX1pjCVFmN0Aw0HggMbevG/daGbVkx1CJzgfOFWN3rDyODUoaF44MOEJQDKNLfck2fJmuAZc14Px3mEVeNKVfU6xFxHhnThOk9Xpzmi4dOnH1uoVtqqSbNMNepNHRiufaS+Ad6CzJBAl3i9Uu2jDevTkHc76zCMyZ1T2e5Esbs27ms98YzW/MyMHF5421peyz7UbwUzP3JhEhEqBH8ubjXKdoysruBU7gI7tXIaQSsh2fFiLsY4M+6+0bHagG1kuskeCudnkSj9P2b7ffheVXagC3CLUTlNzMFt/EMpY="
+   - coverity_scan_run_condition='"$CC" = gcc'
+
+script: ./autogen.sh && make
+
+addons:
+  coverity_scan:
+    project:
+      name: "clearcontainers/shim"
+      description: "Build submitted via Travis CI"
+    notification_email: archana.m.shinde@intel.com
```

Comment:
Until we find a way to allow team notifications in the config file, please could you add the team manually to https://scan.coverity.com/projects/clearcontainers-shim as maintainers/owners? That way, they'll get auto-notified after I run I believe. I've put in a manual request just now to get the ball rolling ;-)